### PR TITLE
feat: add package `@sit-onyx/vitepress-theme`

### LINKED ISSUE #79: Update VitePress and Storybook theme
Update the color theme of our VitePress documentation and Storybook to use the Onyx color palette defined by UX.

### PR DESCRIPTION
**Pull request by larsrickert**
closes #79 

## Description

Add custom Onyx theme for VitePress.

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] All changes are documented in the documentation app (folder `apps/docs`)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

**Comment by BoppLi** [comment]
image

shouldn't we maybe do the digits-mint to green transition here? might look prettier and after all the ownership lays at schwarz IT

**Comment by larsrickert** [comment]
> shouldn't we maybe do the digits-mint to green transition here? might look prettier and after all the ownership lays at schwarz IT

@BoppLi  I talked with Jannick, he said we should leave it like that for now and then we will check again when we have a logo :)

File: packages/vitepress-theme/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  --vp-font-family-base: var(--onyx-font-family);
+  --vp-font-family-mono: var(--onyx-font-family-mono);
+
+  @include mixins.home-page-gradients(
+    var(--onyx-color-brand-primary-500),
+    var(--onyx-color-text-info-500)
+  );
+
+  // remove the following lines when this VitePress PR is merged:
+  // https://github.com/vuejs/vitepress/pull/3434
```
Comment (BoppLi) [inline packages/vitepress-theme/src/index.scss:41]: it is merged ;)

File: packages/vitepress-theme/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  --vp-custom-block-tip-code-bg: var(--vp-c-tip-soft);
+  --vp-code-line-warning-color: var(--vp-c-warning-soft);
+  --vp-code-line-error-color: var(--vp-c-danger-soft);
+}
+
+.dark {
+  // use lighter neutral color in dark mode since its too dark otherwise
+  @include mixins.define-vitepress-color(default, --onyx-color-surface-brand-neutral, 200);
+
+  // VitePress shows a ful white background when hovering in dark mode
```
Comment (BoppLi) [inline packages/vitepress-theme/src/index.scss:54]:
```suggestion
  // VitePress shows a fully white background when hovering in dark mode
```

**Review by BoppLi — APPROVED** [review]

File: packages/vitepress-theme/src/index.scss (hunk 1 above)
Comment (larsrickert) [reply packages/vitepress-theme/src/index.scss:41]: Merged but not release yet :D We need to wait for version `1.0.0-rc.37`

**Review by BoppLi — APPROVED** [review]

File: packages/vitepress-theme/src/index.scss (hunk 1 above)
Comment (larsrickert) [reply packages/vitepress-theme/src/index.scss:41]: Follow up ticket: #117

**Pull request merged**